tests/integration_tests/cpu/propagator/jacobian_validation.cpp

```diff
@@ -437,7 +437,13 @@ struct bound_getter : actor {
             actor_state.m_path_length = stepping._path_length;
             actor_state.m_abs_path_length = stepping._abs_path_length;
             actor_state.m_param_destination = stepping._bound_params;
-            actor_state.m_jacobi = stepping._full_jacobian;
+
+            // Change the bound parameter to the initial state to calculate the
+            // full jacobian
+            propagation._stepping._bound_params = actor_state.m_param_departure;
+            actor_state.m_jacobi =
+                parameter_transporter<algebra_type>().get_full_jacobian(
```

Comment:
So, here, for example, ```get_full_jacobian``` actually also sets the bound track parameters to whatever is currently in the free track parameters of the stepper state, or am I seeing this wrong? If thatt is the case, then I think it might be surprising, if ```propagation._stepping._bound_params``` is not equal to ```actor_state.m_param_departure``` after this call anymore

Reply:
Yeah it gets equal to actor_state.m_param_destination after this call

Reply:
It is not changed anymore